### DISCUSSION TITLE
Is there a varactor in this process?

### DISCUSSION THREAD

**Discussion by hmohajeri** [IHP-GmbH/IHP-Open-PDK#66] [Q&A] [answered]
On the process specification page-13 I do see that there is a s-varicap device which I assume is a varactor. However there is no symbol for it in the released library.

**Comment by KrzysztofHerman** [comment]
Hello @hmohajeri, 
our PDK still contains some devices that neither have symbols nor models and that is because it is still under development.
The feedback from the community helps us to set up a priority list in order to provide the missing pieces. Do you plan any specific application using the mentioned device ?

**Reply by hmohajeri** [reply]
I am not sure what components are missing and a list would be helpful. However, varactors are critical for the design of low noise oscillators and it would be good to have it as soon as possible.

regards,

**Comment by sergeiandreyev** [comment, marked as answer]
Hi @hmohajeri, the `SVaricap` device is now available in the OpenPDK.
